JASP-Engine/JASP/R/reliabilityAnalysisBayesian.R

```diff
@@ -64,19 +64,35 @@ reliabilityBayesian <- function(jaspResults, dataset, options) {
 
 .reliabilityCheckErrors <- function(dataset, options) {
 
-  .hasErrors(dataset = dataset, perform = "run",
+  # check for existing inverse
+  .checkInverse <- function() {
+    if (length(options[["variables"]]) > 2) {
+      use.cases <- "everything"
+      if (any(is.na(dataset))) {
+        if (options[["missingValues"]] == "excludeCasesPairwise") 
+          use.cases <- "pairwise.complete.obs"
+        else if (options[["missingValues"]] == "excludeCasesListwise")
+          use.cases <- "complete.obs"
+      }
+      if (!("matrix" %in% class(try(solve(cov(dataset, use = use.cases)),silent=TRUE)))) {
+        return(gettext("Data covariance matrix is not invertible"))
+      } else {
+        return(NULL)
+      }
+        
+    } else {
+      return(NULL)
+    }
+  }
```

Comment:
```suggestion
      }
    }
    return(NULL)
  }
```
if you make the default return value NULL then you don't need the else statements.